### PR TITLE
Tramstation updates 02/20

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
- Updates various sprites to 3/4 perspective (tram indicator/button, elevator indicator/panel/button)
- Moves button lighting to base typepath
- Renames player facing objects consistently to 'elevator' instead of a mix of 'elevator' and 'lift'
- NTOS theme on elevator panel
- Reduced light power/range of tram indicator sign
- Removed odd wall left in cargo lower floor
- Science elevator panel no longer sitting on a table
- Science keycard auth now directional
- Updates to various mapping helpers
<details>
<summary>Pictures</summary>
  
image
image

</details>
## Why It's Good For The Game

3/4 sprites for when wallening happens. Minor map fixes. Naming consistency.

## Changelog

:cl: LT3
imageadd: New tram and elevator sprites
code: Lighting capability for all buttons
spellcheck: All elevator components are now 'elevator'
code: Reduced brightness on tram platform signs
fix: Fixed odd wall in cargo department
fix: Science elevator panel not laying on a table anymore
fix: Science keycard auth can only be done from RD office
code: More Tramstation mapping helpers
fix: Fixed Icebox pharmacy shutters lighting runtimes
/:cl:

### CODE REVIEW

**Review by MMMiracles — APPROVED** [review]

File: code/game/machinery/buttons.dm [hunk 1]
```diff
@@ -11,6 +11,11 @@
 	var/id = null
 	var/initialized_button = 0
 	var/silicon_access_disabled = FALSE
+	light_power = 0.5 // Minimums, we want the button to glow if it has a mask, not light an area
+	light_range = 1.5
+	light_color = LIGHT_COLOR_DARK_BLUE
+	///The light mask used in the icon file for emissive layer
+	var/light_mask = null
```
Comment (san7890) [inline code/game/machinery/buttons.dm:18]: Move this to be with the other new args.

**Review by san7890 — CHANGES_REQUESTED** [review]

File: code/modules/industrial_lift/tram/tram_machinery.dm [hunk 2]
```diff
@@ -618,10 +618,10 @@ GLOBAL_LIST_EMPTY(tram_doors)
...
 
 /obj/machinery/destination_sign/indicator
-	icon_state = "indicator_central_idle"
+	icon_state = "indicator_off"
 	base_icon_state = "indicator_"
-	light_power = 2
-	light_range = 1.7
+	light_power = 1
+	light_range = 1.5
 	light_color = LIGHT_COLOR_DARK_BLUE
```
Comment (LemonInTheDark) [inline code/modules/industrial_lift/tram/tram_machinery.dm:625]: why override range and color here aren't they already inherited?

File: code/modules/industrial_lift/tram/tram_machinery.dm [hunk 3]
```diff
@@ -620,7 +620,6 @@ GLOBAL_LIST_EMPTY(tram_doors)
 /obj/machinery/destination_sign/indicator
 	icon_state = "indicator_off"
 	base_icon_state = "indicator_"
-	light_power = 1
```
Comment (LemonInTheDark) [inline code/modules/industrial_lift/tram/tram_machinery.dm]: why?

File: code/game/machinery/buttons.dm [hunk 4]
```diff
@@ -66,6 +71,8 @@
 
 /obj/machinery/button/update_overlays()
 	. = ..()
+	if(light_mask && !(machine_stat & (NOPOWER|BROKEN)) && !panel_open)
+		. += emissive_appearance(icon, light_mask, src, alpha = alpha)
```
Comment (LemonInTheDark) [inline code/game/machinery/buttons.dm:75]: alpha won't do anything but it should so just leave it for now

File: code/modules/industrial_lift/elevator_panel.dm [hunk 5]
```diff
@@ -359,4 +364,12 @@
...
-MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/elevator_control_panel, 30)
+/obj/machinery/elevator_control_panel/update_overlays()
+	. = ..()
+	if(!light_mask)
+		return
+
+	if(!(machine_stat & (NOPOWER|BROKEN)) && !panel_open)
+		. += emissive_appearance(icon, light_mask, src, alpha = alpha)
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/elevator_control_panel, 31)
```
Comment (LemonInTheDark) [inline code/modules/industrial_lift/elevator_panel.dm:375]: why 31?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Review by Maurukas — APPROVED** [review]
Maps are good.

**Pull request merged**